### PR TITLE
Feature: Add js plugin sample

### PR DESCRIPTION
**Pull request by Zeroto521**
inspired by https://github.com/duurland/Wox.Plugin.HelloWorldNodeJS

### CODE REVIEW

**Comment by Zeroto521** [comment]
Actually, the js developers are the most people in developers.

So, it's better to pull them to stand our side.

And Flow needs to support the js not in the `exe` method to run the `.js` script.

File: HelloWorldJavascript/plugin.json [hunk 1]
```diff
@@ -0,0 +1,12 @@
+{
+  "ID": "a9dec65b392646c9a8ab55996c930c7b",
+  "ActionKeyword": "hj",
+  "Name": "Hello World Javascript",
+  "Description": "Hello World Javascript",
+  "Author": "durland",
+  "Version": "1.0.0",
+  "Language": "executable",
```
Comment (Zeroto521) [inline HelloWorldJavascript/plugin.json:8]: The Flow needs to support the js not in the `exe` method to run the `.js` script.

**Review by Zeroto521 — COMMENTED** [review]
Actually, the js developers are the most people in developers.

So, it's better to pull them to stand our side.

@Flow-Launcher/team

**Comment by taooceros** [comment]
Seems great, but why not make it native instead of running a bat file?

**Comment by jjw24** [comment]
> Seems great, but why not make it native instead of running a bat file?

happy to merge this, we can do that later

**Pull request merged**